reverse-bits.cpp

```diff
@@ -0,0 +1,51 @@
+// Author: Fatih Baskın
+// Reviewer:
+// Question Link: https://leetcode.com/problems/reverse-bits/
+
+/**
+ * In this question, we have been asked to revert bits of a given number.

```

Comment:
Grammatical error. Not "revert" but "reverse".

Reply:
Ok, done